documentation/docs/05-modules.md

```diff
@@ -27,7 +27,7 @@ import { goto, invalidate, prefetch, prefetchRoutes } from '$app/navigation';
     - `noscroll` (boolean, default `false`) If `true`, the browser will maintain its scroll position rather than scrolling to the top of the page after navigation
     - `keepfocus` (boolean, default `false`) If `true`, the currently focused element will retain focus after navigation. Otherwise, focus will be reset to the body
     - `state` (object, default `{}`) The state of the new/updated history entry
-- `invalidate(href)` causes any `load` functions belonging to the currently active page to re-run if they `fetch` the resource in question. It returns a `Promise` that resolves when the page is subsequently updated.
+- `invalidate(resource, custom)` causes any `load` functions belonging to the currently active page to re-run if they `fetch` or `use` the resource in question. If `custom` is `false` (default) `resource` is a URL that was requested with `fetch`. If `true` resource is a custom identifier registered with `uses(resource)` inside `load`. `invalidate` returns a `Promise` that resolves when the page is subsequently updated.
```

Comment:
maybe `identifier` would be a better name than `custom`?

Reply:
would it be accurate to change it like this?
"any `load` functions belonging to the currently active page" -> "the `load` functions belonging to the currently active page and layouts"